README.md

```diff
@@ -3,7 +3,7 @@
 # GlobalX.Hypermedia.AspNetCore
 
 GlobalX.Hypermedia.AspNetCore transforms your plain old JSON objects into hypermedia aware response. Currently the only
-Using the implemented standard is the [Hal Specification](http://stateless.co/hal_specification.html).
+Using the implemented standard is the [HAL Specification](https://tools.ietf.org/html/draft-kelly-json-hal-00).
 
 The library inspects the Accept header on requests. If set to `application/json+hal` it will transform the object into
```

Comment:
Should this be updated to include that both mime types are valid?

Reply:
Ah right, I missed that.